app/webroot/js/track.js

```diff
@@ -0,0 +1,6 @@
+$("a.github_auth").click(function () {
```

Comment:
__[staging][eslint]__
*  Error - "$" is not defined. (no-undef)

*  Error - Missing "use strict" statement. (strict)